packages/components/psammead-radio-schedule/src/utilities/index.test.js

```diff
@@ -0,0 +1,48 @@
+import detokenise from '@bbc/psammead-detokeniser';
+import durationDictionary from '.';
+
+describe('durationDictionary', () => {
+  it('creates a valid duration dictionary', () => {
+    expect(
+      durationDictionary({ duration: 'PT1H30M', locale: 'en-gb' }),
+    ).toEqual({
+      '%duration%': '1,30,00',
+    });
+  });
+  it('creates a valid duration dictionary and maps token to key correctly', () => {
+    expect(
+      detokenise(
+        'Duration %duration%',
+        durationDictionary({ duration: 'PT1H30M', locale: 'en-gb' }),
+      ),
+    ).toEqual('Duration 1,30,00');
+  });
+  it('creates a valid duration dictionary and returns with specified duration format', () => {
+    expect(
+      detokenise(
+        'Duration %duration%',
+        durationDictionary({
+          duration: 'PT1H30M',
+          separator: ':',
+          locale: 'en-gb',
+        }),
+      ),
+    ).toEqual('Duration 1:30:00');
+  });
+  it('creates a valid duration dictionary but has an invalid string passed in for duration', () => {
```

Comment:
Thanks for adding these tests. However, all of them start with `creates a valid duration dictionary`. I would suggest adding this as a new `describe` block and then leave the reaming text in each of them.